fabric-rendering-v1/src/client/java/net/fabricmc/fabric/api/client/rendering/v1/ArmorRenderer.java

```diff
@@ -47,6 +68,54 @@ static void register(ArmorRenderer renderer, ItemConvertible... items) {
 		ArmorRendererRegistryImpl.register(renderer, items);
 	}
 
+	/**
+	 * Helper method for rendering a {@link TransformCopyingModel}, which will copy transforms from a source model to
+	 * a delegate model when it is rendered.
+	 * @param sourceModel			the model whose transforms will be copied
+	 * @param sourceModelState		the model state of the source model
+	 * @param delegateModel			the model that will be rendered with transforms copied from the source model
+	 * @param delegateModelState	the model state of the delegate model
+	 * @param setDelegateAngles		{@code true} if the {@link Model#setAngles(Object)} method should be called for the
+	 *                              delegate model after it is called for the source model
+	 * @param queue					the {@link RenderCommandQueue}
+	 * @param matrices				the matrix stack
+	 * @param renderLayer			the render layer
+	 * @param light					packed lightmap coordinates
+	 * @param overlay				packed overlay texture coordinates
+	 * @param tintedColor			the color to tint the model with
+	 * @param sprite				the sprite to render the model with, or {@code null} to use the render layer instead
+	 * @param outlineColor			the outline color of the model
+	 * @param crumblingOverlay		the crumbling overlay, or {@code null} for no crumbling overlay
+	 * @param <S>					state type of the source model
+	 * @param <D>					state type of the delegate model
```

Comment:
Nit: the alignment should be done with space chars. This list looks weird and unaligned with any other tab width than 4.

For example, this is how it looks in my github:
<img width="365" height="343" alt="misaligned docs" src="https://github.com/user-attachments/assets/434e3000-30f8-489a-bdb2-afe73da96061" />